Sources/WishKit/SwiftUI/WKButton.swift

```diff
@@ -79,7 +79,7 @@ struct WKButton: View {
     var backgroundColor: Color {
         switch colorScheme {
         case .light:
-            return PrivateTheme.elementBackgroundColor.light
+            return PrivateTheme.systemBackgroundColor.dark
```

Comment:
Doesn't this affect all buttons now? I thought you only wanted to change the Save button style.